iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/infospace/converter/MetadataConverterUtilsTest.java

```diff
@@ -0,0 +1,106 @@
+package eu.dnetlib.iis.wf.importer.infospace.converter;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+import java.time.Year;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.dhp.schema.oaf.DataInfo;
+import eu.dnetlib.dhp.schema.oaf.StructuredProperty;
+import eu.dnetlib.iis.wf.importer.infospace.approver.FieldApprover;
+
+/**
+ * {@link MetadataConverterUtils} test class. 
+ * 
+ * @author mhorst
+ *
+ */
+public class MetadataConverterUtilsTest {
```

Comment:
IDE hint for all method with throws clause: IDE hint: Exception 'java.lang.Exception' is never thrown in the method

Reply:
Fixed all occurrences.